packages/dev/addons/src/htmlMesh/pointerEventsCaptureBehavior.ts

```diff
@@ -5,64 +5,64 @@ import { Logger } from "core/Misc/logger";
 import { requestCapture, requestRelease, releaseCurrent, getCapturingId } from "./pointerEventsCapture";
 
 // Module level variable for holding the current scene
-let _scene: Scene | null = null;
+let _Scene: Scene | null = null;

```

Comment:
There are apparently more than one of these. Is there a linting rule for underscore?

Reply:
Addressed in a different comment